tools/tracegen/TraceGen.cpp

```diff
@@ -58,12 +58,13 @@ help(int argc, char *argv[])
 			|| 0 == strcmp(argv[i], "/h")
 			|| 0 == strcmp(argv[i], "/help")
 		) {
-			printf("%s [-threshold num] [-w2cd] [-generateCfiles] [-treatWarningAsError] [-root rootDir] [-force]\n", argv[0]);
+			printf("%s [-threshold num] [-w2cd] [-generateCfiles] [-treatWarningAsError] [-root rootDir] [-file file.tdf] [-force]\n", argv[0]);
```

Comment:
Ok thanks.  Was just wondering if we should make it fail if you specify both but if it works then I am ok with the current change.